app/code/core/Mage/Catalog/Model/Resource/Eav/Mysql4/Product/Attribute/Backend/Urlkey.php

```diff
@@ -25,7 +25,6 @@
  * @category    Mage
  * @package     Mage_Catalog
  * @author      Magento Core Team <core@magentocommerce.com>
- * @deprecated
```

Comment:
All MySql4-classes are deprecated.

Reply:
should I just remove the change to this file then?

Reply:
Justed marked that line because i'm not sure to keep/remove the other.

Btw ... _there's no mention of why this was done_  ... maybe its not used in core anymore and is just there for BC. For MySql4 classes we could add a notice why they're deprecated.

Reply:
It is used in core (if I'm not mistaked), attribute_code url_key (for entity_type "product") is set to use this class. But also in other places (in the PR description).

Reply:
@sreichel you're right, since all mysql4 classes are deprecated they should stay that way.